### ISSUE TITLE
Lightbox as image block variation

### ISSUE THREAD

**Issue by hanneslsm** [WordPress/gutenberg#55216]
A lot of effort is currently put into the Lightbox feature, which is great. I can't possible see everything thats going on, but I read some PRs and issues. Right now, there are thoughts on how to handle links and where to find settings. In this issue https://github.com/WordPress/gutenberg/issues/55020#issuecomment-1746769100, I had the idea that a image with lightbox could just be a block variation of the Image block. The counterpoint was that block variations look different from each other. 
But after I just saw the draft PR https://github.com/WordPress/gutenberg/pull/55212 I want to bring up this idea again.
I feel like block variations is a powerful tool that could be utilized more to enhance the UX.

Maybe I'm completely wrong. This issue aims to start this discussion.

**Comment by richtabor** [comment]
I'd imagine you'd want to invoke a visual style, say the "Rounded" default variation for the image block, but then if you want it lightbox as well — you can't.

**Comment by mtias** [comment]
I don't think this would work very well in practice. There's an idea to make the lightbox a "behavior" that could be added to different block types.
